### ISSUE TITLE
As an end user I want layout be accessible so that everyone has access

### ISSUE THREAD

**Issue by AliLynne** [AliLynne/tailwind-reakit#5]
AC:
- Research accessible requirements/pitfalls for header sections
    - Add Skip Link above header
- Research accessible requirements/pitfalls for main sections
    - Main section should have id of "main" for skip link
- Research accessible requirements/pitfalls for footer sections
- Add Main, Header, and Footer sections to App.js
